<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>301</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="btn_create">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>229</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>200</y>
      <width>274</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>173</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Offset</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>179</x>
      <y>140</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Random</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_offset">
    <property name="geometry">
     <rect>
      <x>58</x>
      <y>170</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="maximum">
     <double>360.000000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_inverse">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>173</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Inverse</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>76</y>
      <width>274</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_path">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>30</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Select path</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_loops">
    <property name="geometry">
     <rect>
      <x>57</x>
      <y>104</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>10000.000000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>140</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Radius</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_smooth">
    <property name="geometry">
     <rect>
      <x>229</x>
      <y>104</y>
      <width>60</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>3</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>6</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_radius">
    <property name="geometry">
     <rect>
      <x>58</x>
      <y>137</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>107</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Loops</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>181</x>
      <y>107</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Smooth</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_random">
    <property name="geometry">
     <rect>
      <x>229</x>
      <y>137</y>
      <width>60</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_selectPath">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>20</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>path</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progress_Create">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>300</y>
      <width>271</width>
      <height>6</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="format">
     <string>%p%</string>
    </property>
   </widget>
   <zorder>progress_Create</zorder>
   <zorder>btn_create</zorder>
   <zorder>line_3</zorder>
   <zorder>label_5</zorder>
   <zorder>label_4</zorder>
   <zorder>spin_offset</zorder>
   <zorder>check_inverse</zorder>
   <zorder>line</zorder>
   <zorder>ln_path</zorder>
   <zorder>spin_loops</zorder>
   <zorder>label_2</zorder>
   <zorder>spin_smooth</zorder>
   <zorder>spin_radius</zorder>
   <zorder>label</zorder>
   <zorder>label_3</zorder>
   <zorder>spin_random</zorder>
   <zorder>btn_selectPath</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
